src/NNPACK.jl

```diff
@@ -0,0 +1,31 @@
+ccall((:nnp_initialize,"libnnpack"),Void,(),)
+ptp = ccall((:pthreadpool_create, :libnnpack), Ptr{Void}, (Csize_t,), 0)
+
+function softmax!(out::AbstractVecOrMat{T}, xs::AbstractVecOrMat{T}) where T<:AbstractFloat
```

Comment:
Make this more direct – `VecOrMat{Float32}`. I doubt converting to and from is worth it, but if it turns out to be we can use a different overload for that.